openapi/components/schemas/Cashier/CashierCustomPropertySet.yaml

```
@@ -0,0 +1,25 @@
+type: object
+required:
+  - name
+  - properties
+
+properties:
+  id:
+    readOnly: true
+    allOf:
+      - $ref: ../ResourceId.yaml
+  properties:
+    description: |-
+      Defines properties the user can complete when they use the hosted cashier app..
```

nitpick: I think we should avoid referring to cashiers as apps

We have a number of apps within Rebilly, it may confuse users

```suggestion
      Defines properties the user can complete when they use the hosted cashier form.
```